src/applications/coronavirus-chatbot/utils.js

```diff
@@ -22,19 +22,21 @@ const disableButtons = event => {
       : event.target.parentNode;
   const siblingButtons = targetButton.parentNode.childNodes;
 
-  for (let i = 0; i < siblingButtons.length; i++) {
-    siblingButtons[i].disabled = true;
-  }
+  siblingButtons.forEach(button => {
+    const currentButton = button;
+    currentButton.disabled = true;
+    return currentButton;
+  });
```

Comment:
Here's another way of mutating `siblingButtons` without an intermediary variable:
```suggestion
  siblingButtons.forEach((button, index) => {
    siblingButtons[index].disabled = true;
  });
```